commons/src/test/java/com/powsybl/commons/AbstractConverterTest.java

```diff
@@ -52,7 +52,7 @@ public void tearDown() throws IOException {
         fileSystem.close();
     }
 
-    protected static void compareXml(InputStream expected, InputStream actual) {
+    public static void compareXml(InputStream expected, InputStream actual) {
```

Comment:
@miovd  for another PR: move all these static compare* methods to a dedicated utility class like `AssertUtil.compareXml` to avoid using a utility method from an abstract class.
`AbstractConverterTest.compareXml` is not a clean design.

Reply:
I created the issue #2208 related to this comment